pytorch_lightning/callbacks/base.py

```diff
@@ -38,7 +38,7 @@ def state_key(self) -> str:
         """Identifier for the state of the callback.
 
         Used to store and retrieve a callback's state from the checkpoint dictionary by
-        ``checkpoint["callbacks"][state_key]``. Implementations of a callback need to provide a unique state key if 1)
+        ``checkpoint["callbacks"]["state_key"]``. Implementations of a callback need to provide a unique state key if 1)
```

Comment:
I think this was referring to a variable

Reply:
done!